### ISSUE TITLE
Preview not updating properly when going back in history with LCP vignetting correction

### ISSUE THREAD

**Issue by midluk** [Beep6581/RawTherapee#4073]
When I disable or enable the "Vignetting correction" check box in the "Lens Correction Profile" settings the preview properly reflects the change.
When I go back in history, the check box status changes back as expected, but the preview is not reverted.
I have not found any other settings that also have the same problem (LCP "Distortion correction" works properly).

This is with 5.2-113-gd1fe99f8 on Linux x86_64 with current head of "dev" branch.

**Comment by Beep6581** [comment]
Cannot reproduce in `5.2-132-g`ea5f8c3b in `lensfun`
Could you try that branch and report back?

**Comment by midluk** [comment]
The lensfun branch seems to be merged now, but I can still reproduce the problem.

**Comment by Beep6581** [comment]
Confirmed in `5.2-168-g`be2b1ed0

1. Open raw image (`amsterdam.pef`), apply (Neutral)
2. Enable "Lens / Geometry > Profiled Lens Correction > LCP File"
3. Select an LCP file, I used https://filebin.net/j0klla3knby478td/PENTAX__smc_PENTAX-DA_18-55mm_F3.5-5.6_AL__-_RAW.lcp
4. Disable "Vignetting correction", the preview goes a bit darker.
5. Go back in history. The checkbox gets checked but the preview remains the same.
